docs/iot-gateway/integration-with-telegram-bot.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: docwithnav
+title: Integration with Telegram Bot
+
+---
+
+* TOC
+{:toc}
+
+## Overview
+
+Telegram Bots are simply Telegram accounts operated by software – not people – and they'll often have AI features. 
```

Comment:
Please check that there is no copyright statement on the website that contains original version of this text.